cmd/interceptors/main.go

```diff
@@ -92,14 +92,20 @@ func main() {
 		return
 	}
 
-	clusterInterceptorList, err := clusterinterceptorsinformer.Get(ctx).Lister().List(labels.NewSelector())
-	if err != nil {
-		return
-	}
-
-	if err := service.UpdateCRDWithCaCert(ctx, tc.TriggersV1alpha1(), clusterInterceptorList, caCert); err != nil {
-		return
-	}
+	go func() {
+		for {
+			select {
+			case <-time.NewTicker(time.Second).C:
```

Comment:
every second might be too aggressive....maybe every few seconds or so

Reply:
If i keep every few seconds then during initial only it will take time 

So i just added for loop instead of select with everysecond

Reply:
A for loop without a delay would be even more aggressive (and kind of unnecessary) - I think the 1 second delay between refreshes was better.

Ideally, we'd separate out the initial cert update from the one that runs in a loop but we can add that in later

Reply:
Updated 

PTAL 
Thank you